Lottery-Game-Chainlink-VRF.md

```diff
@@ -11,13 +11,13 @@ So we can use some web2 technologies to generate the randomness and then use the
 ## What is an oracle?
 
 - An oracle sends data from the outside world to a blockchain's smart contract and vice-versa.
-- Smart contract can then use this data to make a decision and change its state.
+- Smart contract can then use this data to make a decision and change it's state.
```

Comment:
This should be `its`. You can find more [here](https://www.grammarly.com/blog/its-vs-its/).